views/partials/article-gallery.html.twig

```diff
@@ -0,0 +1,19 @@
+
+<script>
+ var galleryLinks = [];
+
+  {% gimmelist media from articleMedia with {'article': article} %}
+    {% if media.renditions is iterable %}
```

Comment:
there is new way for fetching rendition by name: http://superdesk-web-publisher.readthedocs.io/en/latest/templates_system/article_media.html